app/src/main/java/com/github/sdp/ratemyepfl/database/ReviewsRepository.kt

```diff
@@ -6,4 +6,5 @@ interface ReviewsRepository {
     fun add(value: Review)
     suspend fun get(): List<Review>
     suspend fun getByReviewableId(id: String?): List<Review>
+    suspend fun getPhotosByReviewableId(id: String?): List<Int>
```

Comment:
I don't think this method should be here, we should have a separate Repository for pictures. Maybe I am missing something.

Reply:
Mhh yes, makes sense, as not all Reviewable have photos. I will create a special Repository.